packages/office-ui-fabric-react/src/components/DatePicker/DatePicker.Props.ts

```diff
@@ -117,6 +118,12 @@ export interface IDatePickerProps extends React.Props<DatePicker> {
   showWeekNumbers?: boolean;
 
   /**
+  * Defines when the first week of the year should start, 0, 1 or 2 are the possible values
```

Comment:
> 0, 1 or 2 [](start = 57, length = 10)

Let's be more clear about what 0, 1, and 2 are. This is a public API, we should indicate that the possible values are: FirstWeekOfYear.FirstDay, FirstWeekOfYear.FirstFullWeek, etc